lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,27 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  ## Comparison to `Kernel.quote/2`
+
+  The `escape/2` function is sometimes confused with `Kernel.quote/2`,
```

Comment:
You want `Kernel.SpecialForms.quote/2` in order for it to be linked, not `Kernel.quote/2` :)

Reply:
DOH!